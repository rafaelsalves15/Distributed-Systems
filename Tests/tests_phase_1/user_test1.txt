createAccount A alice
balance A alice
transferTo A broker alice 150
balance A alice
exit
