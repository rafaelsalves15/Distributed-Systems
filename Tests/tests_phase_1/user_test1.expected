UserClientMain
Received 2 arguments
arg[0] = localhost
arg[1] = 2001
> OK
> OK
0
> OK
> OK
150
